modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/Tabs/BasicInfo/BasicInfoTab.tsx

```diff
@@ -216,33 +214,32 @@ export function BasicInfoTab({
 				/>
 			</ContainerWrapper>
 
-			{Liferay.FeatureFlags['LPS-135430'] && !isDefaultStorageType && (
-				<ContainerWrapper
-					collapsable
-					defaultExpanded
-					displayTitle={Liferay.Language.get('external-data-source')}
-					displayType="unstyled"
-					title={Liferay.Language.get('external-data-source')}
-				>
-					<Input
-						className={classNames({
-							'lfr-objects__edit-object-field-model-builder-panel':
-								modelBuilder,
-						})}
-						label={Liferay.Language.get('external-reference-code')}
-						name="externalReferenceCode"
-						onBlur={(event) => {
-							event.stopPropagation();
-
-							if (onSubmit) {
-								onSubmit();
-							}
-						}}
-						onChange={handleChange}
-						value={values.externalReferenceCode}
-					/>
-				</ContainerWrapper>
-			)}
+			<ContainerWrapper
```

Comment:
That's right, removing both `Liferay.FeatureFlags['LPS-135430'] && !isDefaultStorageType` conditions implements the expected behavior.